lib/quantizeAttributes.js

```diff
@@ -0,0 +1,245 @@
+'use strict';
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+
+var addExtensionsUsed = require('./addExtensionsUsed');
+var byteLengthForComponentType = require('./byteLengthForComponentType');
+var getAccessorByteStride = require('./getAccessorByteStride');
+var numberOfComponentsForType = require('./numberOfComponentsForType');
+var packBuffers = require('./packBuffers');
+
+module.exports = quantizeAttributes;
+
+/**
+ * Quantizes attributes in the provided glTF using the WEB3D_quantized_attributes extension.
+ * If options is undefined, all quantizable attributes will be quantized with the default options.
+ * If options.attributes is undefined, all quantizable attributes will be quantized with the provided options.
+ * If options.precision is undefined, the decodeMatrix be written in its full form.
+ *
+ * @param {Object} [gltf] A javascript object holding a glTF hierarchy.
+ * @param {Object} [options=undefined] Defines more specific quantization behavior.
+ * @param {Object} [options.semantics=undefined] Defines which semantics should be quantized.
+ * @param {Number} [options.precision=undefined] Restricts the number of decimal places in the decodeMatrix.
+ *
+ * @returns glTF with quantized attributes
+ *
+ * @example
+ * var gltf = {
+ *     accessors: {
+ *         accessor_0: {...}
+ *         accessor_1: {...}
+ *     },
+ *     meshes: {
```

Comment:
I followed the suggestions in that comment and changed the example; that was part of this round of changes. Instead of having the user specify specifically which attributes in which meshes, it is now just options.semantics: [SEMANTIC1, SEMANTIC2 ...].

A callback is an interesting idea, but I'm not sure it's necessary. I needed to generate corner cases for the reader, in the writer it doesn't really matter, so it's probably better just to keep it simple.